week3/homework/src/todos.json

```diff
@@ -0,0 +1,25 @@
+[
+    {
+        "name": "shopping",
+        "id": "3234-43432",
+        "done" : false
+    },
+    {
+        "name": "visit",
+        "id": "4356-6453",
+        "done" : false
+        
+    },
+    {
+        "name": "call father",
+        "id": "7645-3452",
+        "done" : false
+        
+    },
+    {
+        "name": "shopping",
+        "id": "4343-3523",
+        "done" : false
+        
+    }
+]
```

Comment:
This file shouldn't be committed since it may contain user-sensitive data. I've already described this in #103